src/gwc/src/main/java/org/geoserver/gwc/FakeHttpServletRequest.java

```diff
@@ -300,35 +312,35 @@ public String getRealPath(String arg0) {
     }
 
     public String getRemoteAddr() {
-        return "127.0.0.1";
+        return original.map(r -> r.getRemoteAddr()).orElse("127.0.0.1");
```

Comment:
Wondering, is there a valid situation where the remote address will not be available ? I can't think of any. I would feel more safe throwing a  ``ServletDebugException``. The same opinion for the port (method above).

Reply:
That might break existing code, better to preserve backwards compatibility

Reply:
Ok, fair enough.